azkaban-db/src/main/sql/create.containerization-tables-all.sql

```diff
@@ -0,0 +1,71 @@
+CREATE TABLE IF NOT EXISTS image_types (
+  id               INT         NOT NULL PRIMARY KEY AUTO_INCREMENT,
+  type             VARCHAR(64) NOT NULL UNIQUE,
+  description      VARCHAR(2048),
+  active           BOOLEAN,
+  deployable       VARCHAR(64),
```

Comment:
What can be the value for deployable?
Can you please add index on type and active? It will help with filter.